e2e/map.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { test, expect } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/learn');
+});
+
+test.afterEach(async ({ page }) => {
+  await page.close();
+});
+
+const superBlocksWithLinks = [
```

Comment:
I wonder if we should pull this from the shared config.

Reply:
done